examples/serial_println.rs

```diff
@@ -42,18 +42,21 @@ impl Encoder<String> for LineCodec {
 #[tokio::main]
 async fn main() -> tokio_serial::Result<()> {
     let mut args = env::args();
-    let tty_path = args.nth(1).unwrap_or_else(|| DEFAULT_TTY.into());
+    let tty_path = args.nth(1)
+            .unwrap_or_else(|| DEFAULT_TTY.into());
 
-    let mut port = tokio_serial::new(tty_path, 9600).open_native_async()?;
+    let mut port = tokio_serial::new(tty_path, 115200)
+        .open_native_async()?;
 
     #[cfg(unix)]
-    port.set_exclusive(false)
+    port.set_exclusive(true)
         .expect("Unable to set serial port exclusive to false");
```

Comment:
this expect message should probably be updated to say `true` rather than `false`
```suggestion
        .expect("Unable to set serial port exclusive to true");
```